leaflet.shpfile.js

```diff
@@ -1,8 +1,11 @@
 L.Shapefile =L.GeoJSON.extend({
-    initialize: function (file, options) {
+    initialize: function (file, options, importUrl) {
         if(typeof cw !== 'undefined'){
+            if (typeof importUrl === 'undefined') {
```

Comment:
Yah, that's definitely more defensive and not as overkill as something like `typeof importUrl !== 'string' || importUrl === ''`. I'll go ahead and change it to `importUrl = importUrl || 'shp.js'`. I guess you just have to hope `cw` isn't arbitrarily defined as well, but I'll leave that for a different pr ;p